### PR TITLE
Added page size options in Security events, explore agents

### LINKED ISSUE #2925: Explore agent menu does not allow change row per page
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 4.1.0   | 7.10.0| 4101-1 |

**Description**

It is not possible to change row per page in Explore Agent menu 

**Steps to reproduce**
- Go to `Modules > Security events`
- Select Explore agent
- It is impossible to change `rows per page`

**Screenshots**
row_page_explore_agent_bug

### PR DESCRIPTION
**Pull request by Machi3mfl**
Hi team, this resolves

 - Added page size options in Security events, explore agents table

Closes #2925

### CODE REVIEW

**Review by pablomarga — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Fix restarting selected cluster instead of all of them [#3032](https://github.com/wazuh/wazuh-kibana-app/pull/3032)
 - Fix pinned agents don't trigger a new filtered query [#3035](https://github.com/wazuh/wazuh-kibana-app/pull/3035)
 - Overlay Wazuh menu when Kibana menu is opened or docked [#3038](https://github.com/wazuh/wazuh-kibana-app/pull/3038)
+- Added page size options in Security events, explore agents table [#2925](https://github.com/wazuh/wazuh-kibana-app/pull/2925)
+
```
Comment (Desvelao) [inline CHANGELOG.md:23]: The changelog needs to be updated to `4.2.0-4201`

**Review by gabiwassan — APPROVED** [review]
LGTM

File: server/integration-files/visualizations/cluster/statistics.ts [hunk 2]
```diff
@@ -64,7 +64,7 @@ export default [
     _source: {
       title: 'Wazuh App Statistics remoted tcp sessions',
       visState:
-        `{"title":"Wazuh App Statistics remoted tcp sessions","type":"timelion","params":{"expression":".es(timefield=timestamp,metric=avg:remoted.tcp_sessions, q='*').label(tcp_sessions)","interval":"5m"},"aggs":[]}`,
+        `{"title":"Wazuh App Statistics remoted tcp sessions","type":"timelion","params":{"expression":".es(timefield=timestamp,metric=sum:remoted.tcp_sessions, q='*').label(tcp_sessions)","interval":"5m"},"aggs":[]}`,
```
Comment (frankeros) [inline server/integration-files/visualizations/cluster/statistics.ts:67]: why did you change this metric?
